x-pack/plugins/enterprise_search/public/applications/workplace_search/components/shared/content_section/content_section.tsx

```diff
@@ -30,18 +30,16 @@ export const ContentSection: React.FC<ContentSectionProps> = ({
   description,
   action,
   headerChildren,
-  headerSpacer,
   testSubj,
 }) => (
   <div className={className} data-test-subj={testSubj}>
     {title && (
       <>
         <ViewContentHeader title={title} titleSize="s" description={description} action={action} />
         {headerChildren}
-        {headerSpacer && <EuiSpacer size={headerSpacer} />}
       </>
     )}
     {children}
-    <EuiSpacer size={'xxl'} />
+    <EuiSpacer size={'l'} />
```

Comment:
Can we make this a string? 
```suggestion
    <EuiSpacer size="l" />
```